apps/heft/src/plugins/FileGlobSpecifier.ts

```diff
@@ -35,13 +35,69 @@ export interface IFileSelectionSpecifier {
   includeGlobs?: string[];
 }
 
-export async function getFilePathsAsync(fileGlobSpecifier: IFileSelectionSpecifier): Promise<Set<string>> {
+/**
+ * A supported subset of options used when globbing files.
+ *
+ * @public
+ */
+export interface IPartialGlobOptions {
```

Comment:
Yes. Because we're using a `FileSystemAdapter` passed into the globber, and the  FileSystemAdapter is only partially implemented (doesn't do stuff regarding symlinks, etc.), I only allow passing a specific subset of args that would be useful.

Reply:
I'll update it to be a `Pick<...>` type instead, since all those properties are already optional.